.github/workflows/ci.yml

```diff
@@ -0,0 +1,68 @@
+name: 'Continuous Integration'
+on: [ push, pull_request ]
+jobs:
+  BuildAndPublish:
+    strategy:
+      matrix:
+        alfresco_version: [ 62, 70, 71, 72, 73 ]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out
+        uses: actions/checkout@v3
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          java-version: 11
+          distribution: temurin
+      - name: Login to Docker
+        uses: docker/login-action@v2
+        with:
+          registry: private.docker.xenit.eu
+          username: ${{ secrets.CLOUDSMITH_USER }}
+          password: ${{ secrets.CLOUDSMITH_APIKEY }}
+      - name: Build interface  # Execute before integration testing to catch errors early
+        uses: gradle/gradle-build-action@v2.3.0
+        env:
+          XENIT_ARTIFACTORY_USERNAME: ${{ secrets.XENIT_ARTIFACTORY_USERNAME }}
```

Comment:
Why still use artifactory if we overwrite the commons-lang dep anyways?

Reply:
I think we still use other artifacts... I'll try without.

Reply:
For now we still use acs-packaging that we mirror. We could switch to Alfresco nexus, but I would like to leave that our of scope for this one for now.